test/versions/e2e.spec.ts

```diff
@@ -226,8 +234,19 @@ describe('versions', () => {
       expect(page.url()).toMatch(/\/versions$/)
     })
 
+    test('global - should autosave', async () => {
+      const url = new AdminUrlUtil(serverURL, autoSaveGlobalSlug)
+      // fill out global title and wait for autosave
+      await page.goto(url.global(autoSaveGlobalSlug))
+      await page.locator('#field-title').fill('global title')
+      await wait(1000)
```

Comment:
Could we update the docs, the default called out on the website is `2000ms` but it looks like it should be `800ms`. I was confused how this was auto-saving.

Reply:
Done! I didn't even know the docs said this.